src/XlfFunctions.ts

```diff
@@ -35,7 +35,7 @@ function targetStateActionNeededAsList(): string[] {
     ];
 }
 
-export function GetTargetStateActionNeedeKeywordList(): Array<string> {
+export function GetTargetStateActionNeedKeywordList(): Array<string> {
```

Comment:
GetTargetStateActionNeededKeywordList?

Reply:
btw, it seens to work great! 👍

Reply:
:rofl: I made the wrong spelling correction :hankey: